src/modules/vendor/pages/ni/pedagogy/QuestionnaireProfile.vue

```diff
@@ -46,12 +46,12 @@ export default {
   },
   watch: {
     questionnaire () {
-      this.questionnaireTitle = get(this.questionnaire, 'title') || '';
+      this.questionnaireTitle = get(this.questionnaire, 'title', '');
```

Comment:
@sophiemoustard @cmolle
Que pensez vous de ce changement ? Il me semble qu'on avait eu des soucis avec parce que `get(this.questionnaire, 'title', '')` renvoie null si title est null alors que `get(this.questionnaire, 'title') || ''` renvoie `''`.

Reply:
En effet, il me semble qu'on utilise plus le format `|| ''` maintenant.

Reply:
ça me semblait plus concis, mais je peux remettre comme avant

Reply:
Je suis d'accord avec toi que c'est plus concis, mais comme Kenny l'indique dans son commentaire et comme lodash le precise dans sa doc (https://lodash.com/docs/4.17.15#get) la valeur par defaut n'est prise uniquement dans le cas undefined or on veut qu'elle le soit pour undefined ou null.